flickr_download/flick_download.py

```diff
@@ -317,6 +317,25 @@ def serialize_json(obj):
     return ret
 
 
+def get_photo_sizes(photo):
+    pSize = {}
+    for attempt in range(1, 5):
```

Comment:
define the retry count as a variable on top of the file, so it's not a magic number one has to change in a few places. Add something like `API_RETRIES = 5` on line 34, and then use that variable in this function.

Reply:
Yeah, I also thought about it but then decided to left that _magic numbers_ in this new method only in order to not confuse others with name of constant like `API_RETRIES` because not all api call are retrying. 
Anyway adding new constant as you suggested.